website/pages/docs/configuration/server.mdx

```diff
@@ -65,6 +68,9 @@ server {
   [encryption documentation][encryption] for more details on this option
   and its impact on the cluster.
 
+- `event_buffer_size` `(int: 100)` - Specifies the amount of events generated
```

Comment:
```suggestion
- `event_buffer_size` `(int: 100)` - Specifies the number of events generated
```

Reply:
Should we comment on why one might increase or decrease this value?

Reply:
yeah, how about 
```
- `event_buffer_size` `(int: 100)` - Specifies the number of events generated
  by the server's state store to be held in memory. Increasing this value
  enables new subscribers to have a larger lookback window when initially
  subscribing. Decreasing will lower the amount of memory used for the event
  buffer.
```